packages/fx-core/src/question/create.ts

```diff
@@ -1559,22 +1560,61 @@ export function apiOperationQuestion(includeExistingAPIs = true): MultiSelectQue
     forgetLastValue: true,
     staticOptions: [],
     validation: {
-      validFunc: (input: string[]): string | undefined => {
+      validFunc: (input: string[], inputs?: Inputs): string | undefined => {
         if (input.length < 1 || input.length > 10) {
           return getLocalizedString(
             "core.createProjectQuestion.apiSpec.operation.invalidMessage",
             input.length,
             10
           );
         }
+        const operations: ApiOperation[] = inputs?.supportedApisFromApiSpec as ApiOperation[];
+
+        const authNames: string[] = [];
+        const serverUrls: string[] = [];
+        let firstAuthName;
+        let firstServerUrl;
+        for (const inputItem of input) {
+          const operation = operations.find((op) => op.label === inputItem);
```

Comment:
use "op.id". id is what returned from user inputs.

Reply:
Thanks, will update